scripts/classroom-tool

```diff
@@ -336,3 +412,20 @@ if args.create_report:
     else:
         raise ClassroomToolError(
             "No repositories to report. Did you forget to --create-branches.")
+
+if args.save_feedback:
+    if not os.path.exists("feedback"):
+        logging.info("Creating 'feedback' directory as it does not already exist")
+        os.mkdir("feedback")
+
+    browser = launch_browser()
+    login_to_github(browser)
+
+    roster = pd.read_csv(config("students", "roster"))
+    marking_repo = gh_org().get_repo(config("github", "marking-repo"))
+    prs_by_github_username = {pr.title: pr for pr in marking_repo.get_pulls()}
```

Comment:
I'm a bit confused here. Marking PRs are named by institutional ID, not by GitHub username. How does this bit work?

(Institutional ID is what we want in any event, because the point is to report the PDFs back to the institution).

Reply:
You're right this is confusing. I should be able to simplify it.

Reply:
Done.